.github/workflows/phpunit-tests.yml

```diff
@@ -52,30 +52,18 @@ jobs:
   # - Checks out the WordPress Test reporter repository.
   # - Submit the test results to the WordPress.org host test results.
   test-php:
-    name: ${{ matrix.php }}${{ matrix.multisite && ' multisite' || '' }}${{ matrix.split_slow && ' slow tests' || '' }}${{ matrix.memcached && ' with memcached' || '' }} on ${{ matrix.os }}
+    name: ${{ matrix.php }}${{ matrix.multisite && ' multisite' || '' }}${{ matrix.memcached && ' with memcached' || '' }} on ${{ matrix.os }}
```

Comment:
It could be worth keeping the logic for slow tests around.

Although all the currently defined slow runs are getting removed in this PR, that will change the definition of what is considered slow. There's still some variety of times in the test run on this PR.

https://github.com/WordPress/wordpress-develop/actions/runs/3725663365/usage

Reply:
Isn't that what `git history` is for ? It can be brought back again if needed it in the future.